python/ray/air/_internal/torch_utils.py

```diff
@@ -147,6 +159,9 @@ def convert_ndarray_batch_to_torch_tensor_batch(
     ndarrays: Union[np.ndarray, Dict[str, np.ndarray]],
     dtypes: Optional[Union[torch.dtype, Dict[str, torch.dtype]]] = None,
     device: Optional[str] = None,
+    tensor_creator_fn: Optional[
+        Callable[[np.ndarray, torch.dtype, str], torch.Tensor]
```

Comment:
```suggestion
        Callable[[np.ndarray, torch.dtype, torch.device], torch.Tensor]
```

Reply:
This should have the same type hint as device right? Which is `str`

Reply:
removed this arg